
message CS_MESSAGE_LOGIN
{
	optional	int32	userid		=	1;
	optional	string	password	=	2;
}

message SC_MESSAGE_LOGIN
{
	optional string		answer	=		1;
}